### PR TITLE
Initial deployment & first version

### PR DESCRIPTION
**Pull request by mattzaitley**
- Deployment process with npm publish
- Basic implementation of gallery setup
- Hooks into queue for dynamic insertion/removal of additional galleries

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  }
+  componentWillUnmount() {
+    window.crowdriff__gallery.rendered = window.crowdriff__gallery.rendered.filter(id => id !== `cr__${this.props.hash}`);
+  }
+  shouldComponentUpdate(nextProps) {
+    if (nextProps.allowUpdate) return true;
+    return false;
+  }
+  render() {
+    if (!this.props.hash) console.warn('No gallery hash passed to CrowdRiff gallery component')
```
Comment (BobaFatale) [inline src/index.js:32]: should this return null if there is no hash?

**Review by BobaFatale — COMMENTED** [review]
This is great! just a small question but otherwise everything looks good

File: src/index.js (hunk 1 above)
Comment (mattzaitley) [reply src/index.js:32]: Most of the logic for rendering stuff happens in the componentDidMount anyway - the render just returns a div the component can hook into

Comment (BobaFatale) [reply src/index.js:32]: thank makes sense! thank you

**Review by BobaFatale — APPROVED** [review]

**Pull request merged**